client/packages/system/src/Item/api/hooks/useItems/useItems.ts

```diff
@@ -9,7 +9,7 @@ export const useItems = () => {
 
   return {
     ...useQuery(api.keys.paramList(queryParams), () =>
-      api.get.list(queryParams)
+      api.get.stockItems(queryParams)
```

Comment:
:nit I have a small problem with the hook being `useItems` but it returning only `stockItems` because I'm concerned that later someone will use the hook and be sad that they only get stock items.

A small digression - which I just put in because I'm thinking about it. I'll write up an issue for these ideas..

Since the hook is only used once, we could rename to `useStockItems` though now i'm wondering if the item list shouldn't just show all items and have a `type` column. Probably though, something like mSupply, where we default to showing stock items and have the option to show 'hidden items' 

or.. implement the fancy filter concept which Manu had created, and default the filter to 'stock' items.